jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/ManagementUtils.java

```diff
@@ -788,7 +792,7 @@ static void registerAll() {
 				.validateAndRegister();
 
 			create(ManagementFactory.MEMORY_MANAGER_MXBEAN_DOMAIN_TYPE,
-					// exclude garbage collector beans handled above
+					/* Exclude Garbage Collector beans handled above. */
```

Comment:
"Garbage" and "Collector" don't deserve to be capitalized here.

Reply:
Reverted.